gravitee-apim-console-webui/src/management/api/policy-studio/debug/components/policy-studio-debug-inspector-table/policy-studio-debug-inspector-table.component.html

```diff
@@ -0,0 +1,48 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="policy-studio-debug-inspector-table">
+  <table class="gio-table-light" *ngIf="input">
+    <thead>
+      <tr>
+        <th>key</th>
+        <th>value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr *ngFor="let key of inputKeys">
+        <td>{{ key }}</td>
+        <td>{{ input[key] }}</td>
+      </tr>
```

Comment:
We can use `keyvalue` pipe instead of having to deal with `Object.keys`
```suggestion
      <tr *ngFor="let item of input | keyvalue">
        <td>{{ item.key }}</td>
        <td>{{ item.value }}</td>
      </tr>
```

Reply:
👍